commit-patch-buffer.el

```diff
@@ -152,7 +152,7 @@ following algorithm:
     (beginning-of-buffer)
     (diff-hunk-next) ;; Have to be in a hunk or diff-hunk-file-names won't work.
     (let ((diff-path (reverse (split-string (car (diff-hunk-file-names)) "/")))
-          (file-path (reverse (split-string (buffer-file-name (car (diff-find-source-location))) "/"))))
+          (file-path (reverse (split-string (file-chase-links (buffer-file-name (car (diff-find-source-location)))) "/"))))
```

Comment:
The patch fixed my specific issue, and I didn't go chasing other, similar problems. Following symlinks on both sides of this comparison probably wouldn't hurt, so yeah; I'd do that.